CLI/vault_actions/push.go

```diff
@@ -25,25 +27,39 @@ func LinesInFile(fileName string) []string {
 	return result
 }
 
-func Create_var(data string) Variable {
+func create_var(data string) Variable {
 	var variable Variable
 	variable.Key = strings.Split(data, "=")[0]
 	variable.Value = strings.Split(data, "=")[1]
 	variable.Value = strings.Trim(variable.Value, "\"")
 	return variable
 }
 
-func IsVariableSecureEnvLocal(variable Variable) bool {
+func isVariableSecureEnvLocal(variable Variable) bool {
 	return strings.Contains(variable.Key, "SECURE_ENV_")
 }
 
+func Spush_var(client *vault.Client) *ffcli.Command {
+
+	push := &ffcli.Command{
+		Name:       "push",
+		ShortUsage: "push",
+		ShortHelp:  "Push a file to Vault.",
+		Exec: func(_ context.Context, args []string) error {
+			Push(client)
+			return nil
+		},
+	}
+	return push
+}
+
 func Push(client *vault.Client) bool {
```

Comment:
now it's this function that is exported for nothing imo as it's only used in this file